generate/genhcl/genhcl.go

```diff
@@ -107,6 +94,11 @@ func (h HCL) Condition() bool {
 	return h.condition
 }
 
+func (h HCL) String() string {
+	return fmt.Sprintf("Generating file %q (condition %t) (body %q) (origin %q)",
```

Comment:
```suggestion
	return fmt.Sprintf("generate_hcl %q (condition %t) (body %q) (origin %q)",
```

We are not generating the file here, just a representation of a loaded generate_hcl block.